src/taco-cli/cli/commands.json

```diff
@@ -399,6 +399,9 @@
             ,
             {
                 "name": "--target=TARGET", "description": "[CommandRunTargetDescription]"
+            },
+            {
+                "name":  "--list", "description": "[CommandRunListDescription]"
```

Comment:
Cordova 5.4 provides a DEVICECLASS option for users to specify if they want to list device or emulator targets. You should add that to the command's help description.